selfdrive/car/car_helpers.py

```diff
@@ -95,16 +95,19 @@ def fingerprint(logcan, sendcan):
       cloudlog.warning("Using cached CarParams")
```

Comment:
Could have made these errors, but that would mean we'd have to figure out which log messages correspond to each other. Easy from the logs, not so sure about in Kibana